cmd/juju/resource/formatter.go

```diff
@@ -80,32 +79,39 @@ func FormatSvcResource(res resource.Resource) FormattedSvcResource {
 		CombinedOrigin:   combinedOrigin(used, res),
 		UsedYesNo:        usedYesNo(used),
 	}
+	// Have to check for -1 since revision 0 is still a valid revision.
+	if res.Revision > -1 {
```

Comment:
Is >= 0 clearer than > -1 ?

Reply:
done. I ma not sure if this is better - of course, it adds to readability and maintainability of the code since it caught your eye... I am wondering whether this is a magic number and maybe we should treat revision number check differently...